### PR TITLE
fix: panic send on closed channel in reader pkg

### PR DESCRIPTION
**Pull request by veezhang**
fix: panic send on closed channel in reader pkg

The number of times `statsMgr.StatsCh <- base.NewFileDoneStats(filename)` has reached the number of readers,
then `<-statsMgr.DoneCh` return, but not all readers have exited.
So, it's need to wait for it exit.

### CODE REVIEW

**Comment by veezhang** [comment]
fix #219 #95

File: pkg/cmd/runner.go [hunk 1]
```diff
@@ -76,6 +76,10 @@ func (r *Runner) Run(yaml *config.YAMLConfig) {
 				if err != nil {
 					r.errs = append(r.errs, err)
 					statsMgr.StatsCh <- base.NewFileDoneStats(filename)
+					// fix issues/219
+					// May cause base.FILEDONE to send multiple times, so <-statsMgr.DoneCh return, and then program exit.
+					// It may lead to incomplete import, or even panic.
+					return
```
Comment (yixinglu) [inline pkg/cmd/runner.go:82]: I'm confused about the  purpose of this `return`. Even though without this `return`, this function still always exit when it reaches here.

**Review by yixinglu — COMMENTED** [review]
Could you reproduce the bug reported in these issues? And do you verify the solution?

File: pkg/cmd/runner.go (hunk 1 above)
Comment (veezhang) [reply pkg/cmd/runner.go:82]: Yes, I am solving. It's wip.

**Review by yixinglu — APPROVED** [review]
LGTM

**Pull request merged**